### ISSUE TITLE
多vps可以指定到一个订阅链接吗？

### ISSUE THREAD

**Issue by sjkhsl** [alireza0/s-ui#353]
多vps可以指定到一个订阅链接吗？

**Comment by alireza0** [comment]
Yes, you can use external links and subscription links for each user. It is designed to fetch links from external servers
